gdk-sys/src/lib.rs

```diff
@@ -685,6 +685,7 @@ extern "C" {
     // Gdk Key Handling                                                  NOT OK
     //=========================================================================
     pub fn gdk_keyval_name                         (keyval:c_uint) -> *mut c_char;
+    pub fn gdk_keyval_to_unicode                   (keyval:c_uint) -> u32;
```

Comment:
Yeah, okay, while `u32` is unusual I suppose it does reflect `guint32` more precisely.